templates/default.nginx_http.security.conf.template

```diff
@@ -15,7 +15,7 @@ add_header X-XSS-Protection "1; mode=block" always;
 
 # Enable a referrer policy that protects users' privacy while still enabling
 # Dockstore to see how users interact with the site.
-add_header Referrer-Policy "same-origin" always;
+add_header Referrer-Policy "strict-origin" always;
```

Comment:
It looks like the following setting would achieve the same behavior we currently have on same-origin requests (origin, path, and querystring are sent) while still achieving the safer behavior (origin only being sent) when the request is a cross-origin request.

Note that `strict-origin-when-cross-origin` is the new default behavior as of November 2020 - [link to relevant mozilla developer docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy)

```suggestion
add_header Referrer-Policy "strict-origin-when-cross-origin" always;
```

Reply:
Good catch, updating.